tests/unit/plugins/module_utils/acme/test_backend_openssl_cli.py

```diff
@@ -24,7 +24,6 @@
     ("0000:0001:0000:0000:0001:0000:0000:0001", "0:1::1:0:0:1"),
     ("0000:0001:0000:0001:0000:0001:0000:0001", "0:1:0:1:0:1:0:1"),
     ("0.0.0.0", "0.0.0.0"),
-    ("000.001.000.000", "0.1.0.0"),
```

Comment:
reason?

Reply:
Such IPs are no longer accepted by ipaddress in Python stdlib (see CVE-2021-29921, https://sick.codes/sick-2021-014/).

Reply:
Ah, it'd be good to mention in the commit message or somewhere.

Reply:
730e9e580af8805e1f3b59b1d896fa5a1ace11cc now mentions it.